java/opendcs/src/main/java/decodes/hdb/algo/ParshallFlume.java

```diff
@@ -12,62 +12,52 @@
 // surrogate keys where changed to a DbKey object instead of ject a long/
 import decodes.sql.DbKey;
 import decodes.tsdb.algo.AWAlgoType;
+import org.opendcs.annotations.PropertySpec;
+import org.opendcs.annotations.algorithm.Algorithm;
+import org.opendcs.annotations.algorithm.Input;
+import org.opendcs.annotations.algorithm.Output;
 
 
-//AW:IMPORTS
 import java.lang.Math;
-//AW:IMPORTS_END
 
-//AW:JAVADOC
-/**
-   Computes flow through a Parshall Flume according to the 
-  width property value assigned to the calculation using this
-  algorithm and the logic:
+@Algorithm(description = "Computes flow through a Parshall Flume according to the\n" + 
+   "width property value assigned to the calculation using this\n" + 
+   "algorithm and the logic:\n\n" + 
 
-   If width is greater than 9.0 then:
-   Area = 3.6875 * width + 2.5
-   Flow = Area * (stage ^ 1.6)
+   "If width is greater than 9.0 then:\n" + 
+   "Area = 3.6875 * width + 2.5\n" + 
+   "Flow = Area * (stage ^ 1.6)\n\n" + 
 
-   If width is less than or equal to 9.0 then:
-   Flow = 4 * width * stage ^ (1.522 * (width ^ 0.026)) 
+   "If width is less than or equal to 9.0 then:\n" + 
+   "Flow = 4 * width * stage ^ (1.522 * (width ^ 0.026))\n\n" +  
 
-   Required Properties:  width (ie width=7.0 set in computation property)
+   "Required Properties:  width (ie width=7.0 set in computation property)\n\n" + 
 
-   Required Inputs: stage (observed gauge height)
+   "Required Inputs: stage (observed gauge height)\n\n" + 
 
-   Output variables: flow (Calculated as determined above)
+   "Output variables: flow (Calculated as determined above)\n\n" + 
 
-   Programmed by A. Gilmore July 2008;
-   modified by  M. Bogner  April 2009 to calculate for widths &lt;= 9.0;
-
- */
-//AW:JAVADOC_END
+   "Programmed by A. Gilmore July 2008;\n" + 
+   "modified by  M. Bogner  April 2009 to calculate for widths &lt;= 9.0;")
```

Comment:
The comment about programmer,  should be a regular comment, not part of the annotation.